autopush-common/src/db/bigtable/bigtable_client/mod.rs

```diff
@@ -862,6 +862,15 @@ impl DbClient for BigTableClientImpl {
         Ok(predicate_matched)
     }
 
+    async fn check_user(&self, uaid: &Uuid) -> DbResult<bool> {
```

Comment:
if this feature is ever needed: I think it would be better to move the check into `add_channels` itself. Its write would become a `check_and_mutate` depending on the existence of the row key (the same check used by `add_user` w/ a `true`  instead of `false` filter)